homework-submission/class16-week1/src/components/Todolist.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import Todo from "./Todo";
+
+class Todolist extends Component {
+  render() {
+    const list = [
+      {
+        description: "Get out of bed",
+        deadline: "Wed Sep 13 2017"
+      },
+      {
+        description: "Brush teeth",
+        deadline: "Thu Sep 14 2017"
+      },
+      {
+        description: "Eat breakfast",
+        deadline: "Fri Sep 15 2017"
+      }
+    ];
+    const activity = list.map((item, i) => {
+      return (
+        <Todo key={i} description={item.description} deadline={item.deadline} />
+      );
+    });
+
+    return (
+      <div>
+        <h1 className="todos_header">My Todolist</h1>
+        <ul id="todo_list">{activity}</ul>
```

Comment:
**This is CSS related comment**

Usually, it is not preferable to use `id`, use class instead. classes gives more flexibility to overwrite styles in scalable projects in general.

Read about [CSS selectors specificity](https://css-tricks.com/specifics-on-css-specificity/).

Reply:
Yes, I know, but here as long as I have only one ul so I tended to make it unique. however, I changed it.